config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml

```diff
@@ -54,37 +54,6 @@ periodics:
     testgrid-dashboards: sig-testing-canaries
     testgrid-tab-name: gce
 
-- interval: 2h
-  name: ci-kubernetes-e2e-gce-scalability-canary
-  labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
-    preset-e2e-scalability-common: "true"
-  decorate: true
-  decoration_config:
-    timeout: 140m
-  spec:
-    containers:
-    - command:
-      - runner.sh
-      - /workspace/scenarios/kubernetes_e2e.py
-      args:
-      - --cluster=e2e-big-canary
-      - --extract=ci/latest
-      - --gcp-nodes=100
-      - --gcp-project-type=scalability-project
-      - --gcp-zone=us-east1-b
-      - --provider=gce
-      - --test_args=--ginkgo.focus=\[Feature:Performance\] --gather-resource-usage=true --gather-metrics-at-teardown=true --gather-logs-sizes=true --minStartupPods=8
```

Comment:
We're now using tests from perf-tests repo instead - tests were redesigned and moved around - not removed.

Do we have (and need) canaries for them?
[i.e. should we replace the comamnds here instead of removing?

Reply:
@marseel @mborsz @ameukam

Reply:
I glanced over the per-tests repo and I do not see any tests with the "[Feature:Performance]", but scrapper is still looking for it. Please consider cleaning up as well like here: kubernetes/kubernetes#106283